addons/website/views/snippets/s_quotes_carousel.xml

```diff
@@ -4,70 +4,65 @@
+                    <blockquote class="s_blockquote s_blockquote_classic blockquote me-auto w-50" data-name="Blockquote">
+                        <i class="fa fa-1x fa-quote-left rounded float-start bg-o-color-2 s_blockquote_icon"/>
+                        <div class="s_blockquote_content o_cc1">
+                            <p>Write a quote here from one of your customers. Quotes are a great way to build confidence in your products or services.</p>
+                            <footer class="blockquote-footer">
+                                <img src="/web/image/website.s_quotes_carousel_demo_image_4" class="s_blockquote_avatar img rounded-circle me-2" alt=""/>
+                                <span class="s_blockquote_author text-muted"><b>John DOE</b> &#8226; CEO of MyCompany</span>
+                            </footer>
+                        </div>
+                    </blockquote>
                 </div>
                 <!-- #03 -->
                 <div class="carousel-item oe_img_bg o_bg_img_center pt80 pb80" style="background-image: url('/web/image/website.s_quotes_carousel_demo_image_2'); background-position: 50% 50%;" data-name="Slide">
-                    <div class="container">
-                        <blockquote class="s_blockquote s_blockquote_classic blockquote ms-auto w-50" data-name="Blockquote">
-                            <i class="fa fa-1x fa-quote-left rounded float-start bg-o-color-2 s_blockquote_icon"/>
-                            <div class="s_blockquote_content o_cc1">
-                                <p>Write a quote here from one of your customers. Quotes are a great way to build confidence in your products or services.</p>
-                                <footer class="blockquote-footer">
-                                    <img src="/web/image/website.s_quotes_carousel_demo_image_5" class="s_blockquote_avatar img rounded-circle me-2" alt=""/>
-                                    <span class="s_blockquote_author text-muted"><b>Iris DOE</b> &#8226; CEO of MyCompany</span>
-                                </footer>
-                            </div>
-                        </blockquote>
-                    </div>
+                    <blockquote class="s_blockquote s_blockquote_classic blockquote ms-auto w-50" data-name="Blockquote">
+                        <i class="fa fa-1x fa-quote-left rounded float-start bg-o-color-2 s_blockquote_icon"/>
+                        <div class="s_blockquote_content o_cc1">
+                            <p>Write a quote here from one of your customers. Quotes are a great way to build confidence in your products or services.</p>
+                            <footer class="blockquote-footer">
+                                <img src="/web/image/website.s_quotes_carousel_demo_image_5" class="s_blockquote_avatar img rounded-circle me-2" alt=""/>
+                                <span class="s_blockquote_author text-muted"><b>Iris DOE</b> &#8226; CEO of MyCompany</span>
```

Comment:
The author text looks way too big
<img width="1044" alt="image" src="https://github.com/user-attachments/assets/44ed5c44-624a-4fde-94f8-8b344c7f5ad5">